spec/forms/respondents_detail_spec.rb

```diff
@@ -491,5 +505,38 @@
     end
   end
 
+  context "with a malicious user the remote validator" do
+    it "will not validate the use of a single quote at the start of the string" do
+      populated_respondent_detail.case_number = "'7654321/2017"
+
+      populated_respondent_detail.valid?
+
+      expect(populated_respondent_detail.errors.details[:case_number]).to include a_hash_including(error: :invalid)
+    end
+
+    it "will not validate the use of a single quote at the end of the string" do
```

Comment:
Maybe do the same with double quotes ?  I think the json outputs double quotes by default so would be a good test to make sure it doesnt screw it up